db/migrate/20201016195233_add_last_responded_at_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastRespondedAtToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :clients, :last_response_at, :datetime, null: false, default: -> { 'CURRENT_TIMESTAMP' }
```

Comment:
When you pass a proc as the default, it sets a default for all _existing_ rows. This isn't exactly _the most accurate_ thing, but since we only have a handful of entries in demo and none in prod it seemed simpler and just as (in)accurate as any other value!